doc/Rules.md

```diff
@@ -807,7 +807,7 @@ Aliases: no-duplicate-heading, no-duplicate-header
 
 Parameters: siblings_only, allow_different_nesting (boolean; default `false`)
 
-This rule is triggered if there are multiple headings in the document that have
+This rule is triggered if there are multiple headings in the document that has
```

Comment:
The "have" here refers to the headings (plural), not the document (singular). The original phrasing was correct.